cocotb/share/include/gpi.h

```diff
@@ -43,11 +43,12 @@ Implementation specific notes
 
 By amazing coincidence, VPI and VHPI are strikingly similar which is obviously
 reflected by this header file. Unfortunately, this means that proprietry,
-non-standard, less featured language interfaces (for example Mentor FLI) may have
-to resort to some hackery, or may not even be capable of implementing a GPI layer.
```

Comment:
Where did this comment go?

Reply:
Doesn't seem correct anymore, seeing as an FLI implementation was created.

Reply:
I removed it while cleaning up the mismatched line-splitting clang-format applied, since there is now an FLI implementation.